### DISCUSSION TITLE
Mi institución educativa no aparece en las opciones

### DISCUSSION THREAD

**Discussion by 013bg** [community/community#68144] [GitHub Education] [answered]
### Hi everyone,

Hello everyone, I present the problem that my educational institution with the name "Colegio Nazareth" located in Tuluá, Valle del Cauca, Colombia does not appear among the options of institutions in my municipality, but it does appear located in Microsoft Bing Maps but it does not I know, like the previous time, I was able to make an application attempt without entering one of the options, only the location address and the other requirements. The previous time I did not have the email address they give to teachers, but a teacher agreed to give me his address. email in order to be able to fill out the field in the application but when I made the request I couldn't because it told me that I needed to enter one of the available school options.

Could someone help me by finding a way to not place the option or by sending me the email to communicate with GitHub?

Hola a todos, presento el problema que mi institución educativa  con nombre "Colegio Nazareth" ubicada en Tuluá, Valle del Cauca, Colombia no aparece dentro de las opciones de instituciones en mi municipio mas si en embargo si aparece ubicado en Microsoft Bing Mapas pero no se como la anterior ver pude realizar un intento de solicitud  sin colocar una de las opciones solo la dirección de ubicación y los demás requisitos en la anterior vez no contaba con el correo que le dan a los profesores pero un profesor accedió a darme su dirección de correo con el fin de poder rellenar el campo en la solicitud pero al realizar la solicitud no pude por que me decía que necesitaba colocar una de las opciones de colegios disponibles.

Alguien me podría ayudar buscando una manera de no colocar la opción o pasándome el correo para comunicarme con los de GitHub?

**Comment by tiagoangelos** [comment, marked as answer]
try contacting support:

Email: support@github.com

Page: https://github.com/orgs/community/discussions/categories/github-education?utm_medium=referral&utm_source=liveagent&utm_campaign=directory

HOPE THIS HELPS!
